Tasks/UsePythonVersion/task.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Not critical, but we should probably use a Python logo image with a transparent background.  I can get that for you.